dotcom-rendering/src/palette.ts

```diff
@@ -316,25 +315,134 @@ const starRatingBackgroundColourLight = (): string =>
 const starRatingBackgroundColourDark = (): string =>
 	sourcePalette.brandAlt[200];
 
-const blockQuoteFillLight = (format: ArticleFormat): string => {
-	return decidePalette(format).fill.blockquoteIcon;
+const blockQuoteFillLight = ({ design, theme }: ArticleFormat): string => {
```

Comment:
Think these are already in main from https://github.com/guardian/dotcom-rendering/pull/9469. Not sure why you're getting this diff